LiskHQ/lisk-desktop
# Resolve issue with send token amount validation

### What was the problem?

This PR resolves #5177

### How was it solved?

- Convert amount and balance to BigInt in the validator
- Update unit tests

### How was it tested?

- Navigate to the send form route
- Ensure to have sufficient token balance(possible 20LSK)
- input token amount values of `1`, `3` there should be no form validation error derived.
